src/net/interface_tinygo.go

```diff
@@ -1,3 +1,4 @@
+//go:build tinygo
 // +build tinygo
```

Comment:
Huh, this doesn't make much sense. This is part of the TinyGo source so this build tag is always set. :shrug:

Reply:
Who knows? Seems benign anyhow.